Library/Formula/diff-so-fancy.rb

```diff
@@ -0,0 +1,18 @@
+class DiffSoFancy < Formula
+  desc "builds on the good-lookin' output of diff-highlight"
```

Comment:
I changed the desc to "Good-lookin' diffs with diff-highlight and more" on npm and "Unfancy git-diff" on GitHub